include/plat/pc99/plat/machine/ioapic.h

```diff
@@ -10,6 +10,8 @@
 #include <plat/machine.h>
 #include <arch/types.h>
 
+#define MAX_IOAPIC  (IOAPIC_IRQ_LINES * MAX(1, CONFIG_MAX_NUM_IOAPIC))
```

Comment:
One IOAPIC is allowed if there is no IOAPIC?

Reply:
I don't think seL4 runs on machines  with no IOAPIC.